djangoapp/README.md

```diff
@@ -0,0 +1,12 @@
+# Reference Django app
+Functional skeleton for a Django app that employs secrets management and AWS resources
+Requirements: 
+ * any recent Python 3
+   * macOS: ```brew install python```
+   * CentOS: ```yum -y install epel-release ; yum -y update ; yum -y install python34```
+ * Django itself
+   ```pip3 install --upgrade pip ; pip3 install Django```
+ * AWS SDK
+   ```pip3 install boto3```
```

Comment:
Python dependencies should be defined in a [`requirements.txt`](https://pip.readthedocs.io/en/1.1/requirements.html) file, and installed with `pip3 install -r requirements.txt`, rather than installed individually.

Reply:
done